### ISSUE TITLE
Cannot find module '../build/Debug/nodegit'

### ISSUE THREAD

**Issue by airtonix** [nodegit/nodegit#506]
1. `npm install nodegit --save`
2. `git = require 'nodegit'`
3. launched the script... results in :
```
Error: Cannot find module '../build/Debug/nodegit'
  at Function.Module._resolveFilename (module.js:338:15)
  at Function.Module._load (module.js:280:25)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (DIRNAME/node_modules/nodegit/lib/nodegit.js:12:12)
```

**Comment by tbranyen** [comment]
None of the information you posted is useful for debugging this issue.  What operating system and architecture are you running, what version of Node?

**Comment by tbranyen** [comment]
Updated your comment to remove the non-useful bits.

**Comment by tomruggs** [comment]
In case others have this problem - I experienced this same issue when installing on ubuntu.  It was fixed by updating gcc and g++ to 4.8.  This other issue explains more - https://github.com/nodegit/nodegit/issues/493.
